data/remnant missions.txt

```diff
@@ -493,5 +558,748 @@ mission "Remnant: Technology Available"
+			`	"Ah, <first>! You are just in time," Plume chants quickly, his hands unconsciously dancing in a quick series of gestures. "These cocoons are ready to be delivered, this time to Slylandro. Could you drop them off like you did before?"`
+			choice
+				`	"Yes, I would be happy to bring them home."`
+				`	"Sorry, I'm really not interested in braving the Archon a third time."`
+					decline
+
+			`	"Thank you. Like last time, we can't pay you for this. But we are still hopeful that the Archon might let us resume our studies." He follows you out to <ship> and helps you load the cocoons. Just before parting ways he asks, "Oh, could you record the void sprites this time? We would like to see the behaviors you mentioned last time."`
+				accept
+	npc
+		government " Drak "
+		system Nenia
+		personality staying heroic nemesis uninterested
+		ship "Archon" "Lifted Lorax"
+	on stopover
+		conversation
+			`As you approach Slylandro you watch the Archon like a hawk, but this time it doesn't attempt to drive you off with tridents of white energy. Cautiously you make your way to Slylandro and begin your descent.`
+			`	As <ship> sinks through the cloud layers, you notice the void sprites seem to be tailing you. A few drop past you with startling speed, only to reappear from below as you approach the designated altitude. At the last minute you remember Plume's request to record them, and hit the switch to activate the recorder for the exterior cameras.`
+			`	Setting the ship on auto-pilot, you head back and suit-up in the bulky pressure suit once again. Readying the eggs in the cargo hold, you equalize pressure and open the cargo bay doors. As before, the eggs are already floating as you open the cocoons and gently carry them to the door.`	
+			`	Almost as soon as you release the first egg, a void sprite is there, brushing against the ship to scoop the egg away into the swirling mists. By the time you return with the next one, another sprite is already waiting at the door, undulating in the eerie light. By the time the last egg is released you can see dozens of the strange creatures swarming around the collection of eggs. It occurs to you that very few humans would ever have the chance to see anything like this...`
+			`	After taking a moment to enjoy the view, you shut the door as gently as you can, purge the ship with fresh air, and begin the journey back to Aventine.`
+	on complete
+		"reputation:  Drak " += 1
+		conversation
+			`Back on Aventine, you share the recording of the void sprite's behavior with Plume. "These Sprites demonstrate considerably more communication than we originally thought. And obviously they must have communicated what you did on Nasqueron. Interesting..." Plume's excited staccato exclamations trail off as he begins to make notes on a datapad he has on hand.`
+
+		
+mission "Remnant: Learning Sign 1"
```

Comment:
That's a good idea. Right now I think it would needlessly slow things down, since most of the rest of the missions are written assuming the player has some linguistic ability and there really isn't much else to do in Remnant space, but once more missions get added I think changing this would be good to do.

I suspect that if the player had to spend a month jumping around doing nothing before finishing that and getting to go on to more missions would probably have a lot of players thinking that they'd either a) missed something, or b) hit the end of where missions are and are done. 

I don't see any beneficial reason for forcing the player to force the clock forward while there isn't stuff for them to do to move it forward organically. That being said, I'll definitely have this on my list of things to change once there is more content.